group_project_v2/public/css/group_project_dashboard.css

```diff
@@ -246,6 +246,11 @@ table.activity-data tr.user-data-row {
     display: none;
 }
 
+table.activity-data tr.user-data-row.filtered-out td a
+{
+    color: gray;
```

Comment:
@jbzdak two nits here:
1. Open parenthesis on the same line.
2. McKinsey uses some particular "gray" color: `#868685`. I'd suggest using it right away, or they will notice that and return it as a bug.